Optimizing.py

```diff
@@ -0,0 +1,109 @@
+##########################################
+# Optimizing Model Parameters
+
+import torch
+from torch import nn
+from torch.utils.data import DataLoader
+from torchvision import datasets
+from torchvision.transforms import ToTensor, Lambda
+
+training_data = datasets.FashionMNIST(
+    root="data",
+    train=True,
+    download=True,
+    transform=ToTensor()
+)
+
+test_data = datasets.FashionMNIST(
+    root="data",
+    train=False,
+    download=True,
+    transform=ToTensor()
+)
+
+train_dataloader = DataLoader(training_data, batch_size=64)
+test_dataloader = DataLoader(test_data, batch_size=64)
+
+class NeuralNetwork(nn.Module):
```

Comment:
So this is a good linear split, but you see how we're duplicating code across files? How do we avoid that, and actually "import" code blocks from other files?

Let's talk about this today!